src/components/pipeline-warning/pipeline-warning.js

```diff
@@ -47,10 +47,12 @@ export const PipelineWarning = ({
             a while to render. You can use the sidebar controls to select a
             smaller graph.
           </p>
-          <Button onClick={onHide}>Render it anyway</Button>
-          <Button mode="secondary" onClick={onDisable} size="small">
-            Don't show this again
-          </Button>
+          <div className="pipeline-warning__button-wrapper">
```

Comment:
hey @rashidakanchwala  Do we not need any styling for this new classname ?

Reply:
nope, this div basically ensures that the buttons are not next to each other ... just followed this way (https://github.com/kedro-org/kedro-viz/blob/main/src/components/settings-modal/settings-modal.js#L149)